examples/ansibleee-playbook-local.yaml

```diff
@@ -4,18 +4,8 @@ metadata:
   name: ansibleee-playbook-local
   namespace: openstack
 spec:
-  # We can specify either playbook, which will run with default ansible-runner options,
-  # or args, which allows specify the whole command that we want to execute
-  # play: |
-  #   - name: Print hello world
-  #     hosts: all
-  #     tasks:
-  #     - name: Using debug statement
-  #       ansible.builtin.debug:
-  #         msg: "Hello, world!"
   playbook: "test.yaml"
   # args: ["ansible-runner", "run", "/runner", "-vvv", "-p", "test.yaml"]
-  image: "quay.io/jlarriba/openstack-tripleo-ansible-ee:stable"
```

Comment:
`extraMounts` of this file is missing a "-" (and the CR isn't created if you try `oc create -f `).

You have to change L21 from `extraVolType: Ceph` to `- extraVolType: Ceph` because it's a list

Reply:
Done